design-proposals/klip-42-schema-migrations-tool.md

```diff
@@ -271,9 +267,7 @@ The `MIGRATION_EVENTS` stream output:
 +-------------+---------+---------------+----------+------------+---------------------+---------------------+----------+
 | 1           | 1       | Initial setup | Migrated | <MD5-sum>  | 12-01-2020 03:48:00 | 12-01-2020 03:48:05 | null     |
```

Comment:
jsut to make sure I understand correctly, the stream should also have as it's second row, right?
```
 CURRENT     | 1       | Add users     | Migrated | <MD5-sum>  | 12-01-2020 03:48:00 | 12-01-2020 03:48:05 | 1        |
```

Reply:
Oh yes, that one should be present in the stream as well. I updated the klip.